components/org.wso2.carbon.business.rules.web/src/utils/AuthManager.jsx

```diff
@@ -36,13 +36,74 @@ const appContext = window.contextPath;
  * Authentication manager.
  */
 export default class AuthManager {
+
+
+
```

Comment:
Is this auto generated ? else shall we remove the new lines

Reply:
This has been already fixed. _(See https://github.com/wso2/carbon-analytics/blob/master/components/org.wso2.carbon.business.rules.web/src/utils/AuthManager.js#L39)_.